src/test-resources/basic-axe-v3.2.2-reporter-v2.json

```diff
@@ -14,7 +14,7 @@
         "orientationType": "landscape-primary"
     },
     "timestamp": "2019-03-22T19:12:06.129Z",
-    "targetPageUrl": "https://www.washington.edu/accesscomputing/AU/before.html",
+    "url": "https://www.washington.edu/accesscomputing/AU/before.html",
     "targetPageTitle": "Accessible University Demo Site - Inaccessible Version",
```

Comment:
We should probably remove `targetPageTitle` too, if Axe doesn't include it

Reply:
Oops, I missed that part. Thanks!